### PR TITLE
Thellimist/pi 954

### PR DESCRIPTION
**Pull request by thellimist**

### CODE REVIEW

File: src/Cpanel/ClientActions.php [hunk 1]
```diff
@@ -143,15 +143,14 @@ public function addSSLVerficiationDNSRecordForCName($zoneList)
                         $dnsRecord->setContent($recordTarget);
                         $dnsRecord->setTtl(14400);
 
-                        return $this->cpanelAPI->addDNSRecord($zoneName, $dnsRecord);
+                        $this->cpanelAPI->addDNSRecord($zoneName, $dnsRecord);
```
Comment (jwineman) [inline src/Cpanel/ClientActions.php]: Why was this removed?  This is how you tell whether the record was added successfully or not.

Comment (thellimist) [inline src/Cpanel/ClientActions.php]: There can be multiple records being added.
